### ISSUE TITLE
'Selector' option throws TypeError

### ISSUE THREAD

**Issue by SWBennett06** [postcss/postcss-calc#39]
With the 'selector' option set to true using 6.0.0, the plugin throws the following error. This occurs even without any selectors being built with calc() in my stylesheets. 

```
TypeError: Cannot read property 'charCodeAt' of undefined
  at module.exports (fakepath/node_modules/postcss-value-parser/lib/parse.js:17:21)
  at new ValueParser (fakepath/node_modules/postcss-value-parser/lib/index.js:7:22)
  at ValueParser (fakepath/node_modules/postcss-value-parser/lib/index.js:10:12)
  at exports.default (fakepath/node_modules/postcss-calc/node_modules/reduce-css-calc/dist/index.js:29:43)
  at transformValue (fakepath/node_modules/postcss-calc/dist/lib/transform.js:20:45)
  at fakepath/node_modules/postcss-calc/dist/lib/transform.js:36:19
  at fakepath/node_modules/postcss-selector-parser/dist/selectors/container.js:170:26
  at Selector.each (fakepath/node_modules/postcss-selector-parser/dist/selectors/container.js:153:22)
  at Selector.walk (fakepath/node_modules/postcss-selector-parser/dist/selectors/container.js:169:21)
  at fakepath/node_modules/postcss-selector-parser/dist/selectors/container.js:173:31
```
Running through the gulp-postcss 7.0.0, which is using PostCSS 6.0.0

**Comment by mjschlot** [comment]
The issue appears to be related to selectors like:
`abbr[title]`
This situation exists in packages like normalize.css (https://github.com/necolas/normalize.css/blob/master/normalize.css).

Possible fix would be to test if node.value is undefined in transform.js
`if (node.type === 'attribute' && node.value !== undefined) {`

I'm not familiar enough with postcss to know if this is the best fix though.

**Comment by wenkanglin** [comment]
I has same issue. Does anyone know how to solve this problem ?

**Comment by mjschlot** [comment]
@wenkanglin  I know of two options:
- Avoid the use of attribute selectors that do not have a value (e.g. abbr[title])
- Or, the code needs to be updated like I suggested in my last post. I would submit a pull request but I really don't know if my fix the best way to address this or not.

**Comment by wenkanglin** [comment]
Thank you for answer. The first option is correct fix the issue for me. But we often inevitably encounter code like below:
```
button[disabled] {}
```
@mjschlot you should submit a pr and wait merged by author. :+1:

**Comment by Semigradsky** [comment]
Fixed in v6.0.1
